src/api/schemas.py

```diff
@@ -157,7 +179,8 @@ def specializations_str_validation(cls, value: str):
             new_value = [int(value) for value in value.split(", ")]
             return new_value
         except ValueError:
-            raise ValueError('Для передачи строки с числами в поле specializations используйте формат: "1, 2, 3" ')
+            raise ValueError(
+                'Для передачи строки с числами в поле specializations используйте формат: "1, 2, 3" ')
```

Comment:
Почему тут поправил и выше тоже? Прошлые строки ни как не противоречили нашему code-style, а теперь есть немного)  
```suggestion
            raise ValueError(
                'Для передачи строки с числами в поле specializations используйте формат: "1, 2, 3"'
            )
```

Reply:
да не правил я... не пойму ничего.